cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -22,11 +22,17 @@
 
 
 @pytest.mark.parametrize("navigation", ["provider_vms", "all_vms", "vm_summary"])
+@pytest.mark.meta(blockers=[BZ(1717483, forced_streams=["5.11"])])
 @pytest.mark.uncollectif(
-    lambda navigation: navigation == "provider_vms" and (BZ(1686617).blocks or BZ(1686619).blocks),
+    lambda navigation, appliance: (
+        navigation == "provider_vms" and
+        (appliance.version < "5.11" or
+         BZ(1717539).blocks)
+    ),
     reason="This test is blocked by the following BZs: {}, {}".format(
         BZ(1686617).bug_id,
-        BZ(1686619).bug_id
+        BZ(1686619).bug_id,
```

Comment:
This is no longer true for this uncollect statement - these BZ's are no longer included in the lambda.